components/hab/src/command/plan/init.rs

```diff
@@ -107,19 +92,11 @@ pub fn start(ui: &mut UI,
     }
 
     // We want to render the configured variables.
-    if windows {
-        if with_all || scaffold.is_none() {
-            let rendered_plan = handlebars.template_render(FULL_PLAN_PS1_TEMPLATE, &data)?;
-            create_with_template(ui, &format!("{}/plan.ps1", root), &rendered_plan)?;
-        } else {
-            let rendered_plan = handlebars.template_render(DEFAULT_PLAN_PS1_TEMPLATE, &data)?;
-            create_with_template(ui, &format!("{}/plan.ps1", root), &rendered_plan)?;
-        }
-    } else if with_all || scaffold.is_none() {
-        let rendered_plan = handlebars.template_render(FULL_PLAN_TEMPLATE, &data)?;
-        create_with_template(ui, &format!("{}/plan.sh", root), &rendered_plan)?;
+    if cfg!(windows) {
+        let rendered_plan = handlebars.template_render(FULL_PLAN_PS1_TEMPLATE, &data)?;
+        create_with_template(ui, &format!("{}/plan.ps1", root), &rendered_plan)?;
     } else {
-        let rendered_plan = handlebars.template_render(DEFAULT_PLAN_TEMPLATE, &data)?;
+        let rendered_plan = handlebars.template_render(FULL_PLAN_TEMPLATE, &data)?;
```

Comment:
Wonder if we should take this opportunity to rename this to `FULL_PLAN_BASH_TEMPLATE` instead.